includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -169,6 +170,37 @@ public static function add_admin_hooks() {
 		} );
 	}
 
+	/**
+	 * Enqueue style.
+	 */
+	public static function enqueue_admin_assets() {
```

Comment:
Can this name be made more specific? There is `enqueue_edit_post_screen_scripts`. So should it be `enqueue_post_list_screen_scripts`?